src/AddTodoForm.jsx

```diff
@@ -14,10 +14,7 @@ function AddTodoForm({ onAddTodo }) {
     // Function to handle form submission
     const handleAddTodo = (event) => {
         event.preventDefault();
-        const newTodo = {
-            id: Date.now(),
-            title: todoTitle,
-        };
+        const newTodo = todoTitle
```

Comment:
now this line is unnecessary, you can put todoTitle  in onAddTodo function: onAddTodo(todoTitle);